plugins/lua/blueprint.lua

```diff
@@ -43,17 +44,11 @@ local valid_phase_list = {
 valid_phases = utils.invert(valid_phase_list)
 
 local function parse_cursor(opts, arg)
-    local _, _, x, y, z = arg:find('^(%d+),(%d+),(%d+)$')
-    if not x then
-        qerror(('invalid argument for --cursor option: "%s"; expected format' ..
-                ' is "<x>,<y>,<z>", for example: "30,60,150"'):format(arg))
-    end
+    local cursor = argparse.coords(arg)
```

Comment:
I'm getting a test failure locally on this branch (when merged into develop @ f1ec222d9a6ec910af271731cdc71c0815d6678e) that I think is due to this change:
```
error: specified coordinates not on current map: "%s": 1,2,3

test errored: plugins/blueprint:parse_gui_commandline
```
Unfortunately it lacks a traceback, but there's only one test it could refer to:
```
    opts = {}
    b.parse_gui_commandline(opts, {'--cursor=1,2,3'})
    expect.table_eq({auto_phase=true, name='blueprint', start={x=1,y=2,z=3}},
                    opts)
```

Reply:
I see why I missed this. `argparse.coords()` checks the validity of the coordinates, which will fail if a map is not loaded. the tests are marked as `mode=none` so a map is not guaranteed to be loaded, but it just so happened to be loaded every time I ran the tests.

CI would have caught this once I merged in the argparse PR, though, so this doesn't make me concerned that the test system is fragile or anything. Fixed.

Also fixed error string formatting error in argparse.

Reply:
Yep, I merged in the argparse PR (#1897) in an attempt to get CI to do this, but re-running CI didn't make it pick up the new commits from develop (only rebasing did, it seems).

Thanks for the formatting fix as well!